packages/docs/docs/02-Guides/08-Sign in with Ethereum.mdx

```diff
@@ -0,0 +1,56 @@
+# Sign in with Ethereum
+
+**Prerequisites**
```

Comment:
Use headers with `#`. This bold text does not allow anchor linking and the outline does not show up in the panel on the right.